facebook/delphiFacebook/R/responses.R

```diff
@@ -193,6 +193,7 @@ load_response_one <- function(input_filename, params, contingency_run) {
   input_data <- code_activities(input_data, wave)
   input_data <- code_vaccines(input_data, wave)
   input_data <- code_schooling(input_data, wave)
+  input_data <- code_children(input_data, wave)
```

Comment:
this function name sounds ominous

Reply:
<img width="95" alt="image" src="https://user-images.githubusercontent.com/1158666/157698034-74641677-536b-4462-9d7f-3de5a2478ade.png">